app/docs/enterprise/0.31-x/installation/docker.md

```diff
@@ -55,21 +55,31 @@ A guide to installing locally Kong Enterprise Edition (and its new license file)
           -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
           -e "KONG_VITALS=on" \
           -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+          -e "KONG_PORTAL=on" \
+          -e "KONG_PORTAL_API_URI=0.0.0.0:8000/portal" \
```

Comment:
this api_url assumes that the user will create an API (which is needed only for authentication). Maybe leave off of base install?

Reply:
just saw this. any harm in leaving it in even if unused for now?

Reply:
deal